index.js

```diff
@@ -0,0 +1,25 @@
+import Book from './modules/books.js';
+import { contactPage, booksList, addBook } from './modules/toggleDisplay.js';
+
+const booksLibrary = new Book();
+booksLibrary.getBookList();
+const formData = document.getElementById('new-book');
+formData.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const getTitle = document.getElementById('title').value;
+  const getAuthor = document.getElementById('author').value;
+  booksLibrary.addNewBook(getTitle, getAuthor);
+});
```

Comment:
- [ ] **[OPTIONAL]** For a better user experience, it would be nice to clear the inputs after submission.